### PR TITLE
helm: added scheduler requeue sleep environment variable

### PR DESCRIPTION
**Pull request by audrium**
closes https://github.com/reanahub/reana-server/issues/382

### CODE REVIEW

File: helm/configurations/values-dev.yaml [hunk 1]
```diff
@@ -3,9 +3,6 @@
 components:
     reana_server:
       image: reanahub/reana-server
-      environment:
-        REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY: 60
-        REANA_SCHEDULER_SECONDS_RETRY_DELAY: 6
```
Comment (tiborsimko) [inline helm/configurations/values-dev.yaml:8]: Not setting anything explicit for the DEV setup here?

**Comment by mvidalgarcia** [comment]
Works well, needs rebase.

File: helm/configurations/values-dev.yaml (hunk 1 above)
Comment (mvidalgarcia) [reply helm/configurations/values-dev.yaml:8]: Yeah, it's not nice to wait 15 secs in between `helloworld` runs when in dev mode.

Comment (audrium) [reply helm/configurations/values-dev.yaml:8]: I have added 2 seconds for DEV setup. After testing with balanced strategy it seems to be enough, works in the same way as 15 seconds

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**